GDJS/Runtime/pixi-renderers/pixi-image-manager.ts

```diff
@@ -95,7 +95,7 @@ namespace gdjs {
     setResources(resourceDataArray: ResourceData[]): void {
       this._resources.clear();
       for (const resourceData of resourceDataArray) {
-        if (resourceData.kind === 'image') {
+        if (resourceData.kind === 'image' || resourceData.kind === 'video') {
```

Comment:
Is there another resource kind we might have forgotten?

Reply:
I checked all, and this was the only missing